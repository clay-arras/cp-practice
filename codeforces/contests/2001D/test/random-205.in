1
15
19 12 19 5 9 5 22 12 7 16 15 10 3 22 15
