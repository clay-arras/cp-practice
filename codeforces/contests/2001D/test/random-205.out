10
19 5 9 12 7 16 15 10 3 22
